projects/valtimo/dossier/src/lib/dossier-detail/dossier-detail.component.html

```diff
@@ -102,8 +102,8 @@
       }"
       *ngFor="let item of customDossierHeaderItems"
     >
-      <span *ngIf="item.label !== ''">{{ item.label | translate }}</span>
-      <span *ngIf="item.label !== '' && item.value !== '' && item.value !== null">: </span>
+      <span *ngIf="item.label != ''">{{ item.label | translate }}</span>
```

Comment:
Why !== to != ? It's best to use !== && === - https://medium.com/swlh/javascript-best-practices-variables-and-comparisons-afef97ec01b7. If you want to check if they have a value, just do `*ngIf="item.label"` and `*ngIf="item.label && item.value"`

Reply:
Ah alright, thanks, yeah it wasn't working with !== for some reason, I'll try your approach : )